src/containers/App/App.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import Header from "../../components/Header/Header";
```

Comment:
If I were to be really picky, please add a line break between package imports and app imports.